packages/components/src/components/hds/copy/button/index.ts

```diff
@@ -70,23 +85,25 @@ export default class HdsCopyButtonComponent extends Component {
   }
 
   @action
-  onSuccess(args) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onSuccess(args: any) {
     this.status = 'success';
     this.resetStatusDelayed();
 
-    let { onSuccess } = this.args;
+    const { onSuccess } = this.args;
 
     if (typeof onSuccess === 'function') {
       onSuccess(args);
     }
   }
 
   @action
-  onError(args) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onError(args: any) {
```

Comment:
@alex-ju would this be correct?
```suggestion
  onError(args: ModifierSignature['Args']['Named']['onError']) {
```

Reply:
yes, same as above